minard/templates/runselection.html

```diff
@@ -0,0 +1,93 @@
+                {% endif %}
+            </div>
+            <div class="col-md-10 text-right">
+                Result:
+                <select onchange="location=this.options[this.selectedIndex].value;">
+                    {% for i, text in [("All", "All"), ("Pass", "Pass"), ("Fail", "Fail"), ("Purgatory", "Purgatory")] %}
+                        <option {% if result == i %}selected="selected" {% endif %}value="{{url_for("runselection", limit=limit, offset=offset, criteria=criteria, result=i)}}">{{ text }}</option>
+                    {% endfor %}
+                </select>
+            </div>
+            <div class="col-md-10 text-right">
+                Criteria:
+                <select onchange="location=this.options[this.selectedIndex].value;">
+                    {% for i, text in [("scintillator", "scintillator"), ("partial_fill_antinu", "partial_fill_antinu"), ("partial_fill", "partial_fill"), ("water", "water")] %} <!-- FIXME: make this dynamic !-->
+                        <option {% if criteria == i %}selected="selected" {% endif %}value="{{url_for("runselection", limit=limit, offset=offset, criteria=i, result=result)}}">{{ text }}</option>
+                    {% endfor %}
+                </select>
+            </div>
+            <div class="col-md-10 text-right">
+                Runs per page:
+                <select onchange="location=this.options[this.selectedIndex].value;">
+                    {% for i, text in [(10,"10"), (25, "25"), (50, "50"), (100, "100")] %}
+                        <option {% if limit == i %}selected="selected" {% endif %}value="{{url_for("runselection", limit=i, offset=offset, criteria=criteria, result=result)}}">{{ text }}</option>
+                    {% endfor %}
+                </select>
+            </div>
+        </div>
+        <div class="row">
+            <div class="col-md-12">
+                <table class="table table-hover">
+                    <thead>
+                        <tr>
+                            <th>Run</th>
+                            <th>Result</th>
+                            <th>Criteria</th>
+                            <th>Overlord</th>
+                            <th>Last review</th>
+                        </tr>
+                    </thead>
+                    <tbody>
+                        {% if run_info == False %}
+                            <tr class="bg-danger" >
+                                <td>
+                                    {{"N/A"}}
+                                </td>
+                                <td colspan="10" style="text-align:center;">
+                                    <b> No RS results present for selected range. </b>
+                                </td>
+                            </tr>
+                        {% else %}
+                            {% for run_number in run_info.keys() %}
```

Comment:
It would be good to try and order by run-number descending for this table. 
![run_order](https://user-images.githubusercontent.com/2319312/210612078-55104f05-50dd-4f84-b4f6-4a569690c760.png)